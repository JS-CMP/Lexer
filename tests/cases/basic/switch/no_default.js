var value = "z";
switch (value) {
    case "a":
        x = 1;
        break;
    case "b":
        x = 2;
        break;
}

