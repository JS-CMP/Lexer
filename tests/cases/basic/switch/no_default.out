// not handle by the lexer for now
#include <iostream>
int main() {std::cerr << "somethings was not handle" << std::endl; return 0;}